pallets/genetic-testing/src/lib.rs

```diff
@@ -460,7 +409,7 @@ impl<T: Config> GeneticTestingInterface<T> for Pallet<T> {
 
     // ------------------------ Update ---------------------
 
-    fn process_dna_sample(lab_id: &T::AccountId, tracking_id: &Vec<u8>) -> Result<Self::DnaSample, Self::Error> {
+    fn process_dna_sample(lab_id: &T::AccountId, tracking_id: &Vec<u8>, _status: Self::DnaSampleStatus) -> Result<Self::DnaSample, Self::Error> {
```

Comment:
"_status"?

Reply:
leading _ nya di hapus saja ya

Reply:
If I remove the underscore, I got this warning. Is it okay to proceed?
![Screenshot from 2021-07-23 08-15-26](https://user-images.githubusercontent.com/72794806/126724321-3f5f91e3-48f0-47b9-8949-7392a097c2c3.png)

Reply:
Nah itu error karena variable status gak di pake bli, di bawah"nya aku perhatiin masih ada bagian yang di hardcode haha @SHA888

Reply:
O iya ya... Okay aku cobar perbaikin dulu ya. Thank you mas @agustinustheo